normalizr/regex.py

```diff
@@ -1,3 +1,5 @@
+#-*- coding: utf-8 -*-
```

Comment:
I think is not really needed. Anyway, you can create a PR to be discussed.

Cheers,
David